src/libs/routes/notFoundRoute.ts

```diff
@@ -0,0 +1,7 @@
+let notFound: any = (req: any, res: any, next: any) => {
+  const err = 'Not Found';
+  res.status(404).send(err);
```

Comment:
- notFoundRoute will not response anything.
- It will call errorHandler with some message.